build.gradle

```diff
@@ -96,6 +92,70 @@ repositories {
     maven { url "https://plugins.gradle.org/m2/" }
     maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/" }
 }
+configurations {
+    zipArchive
+    testImplementation {
+        exclude group: 'org.hamcrest', module: 'hamcrest-core'
```

Comment:
Why do you need to exclude hamcrest?

Reply:
It was already there, I just moved this from later in the file. The reason we originally have it is cause without excluding it we have a jarHell Exception caused by some conflict of hamcrest in jdk11. It was added in the initial release PR for AD in opendistro.

Reply:
could you add a comment?

Reply:
added